.github/workflows/integration-tests.yml

```diff
@@ -16,8 +16,6 @@ jobs:
       uses: mamba-org/provision-with-micromamba@main
       with:
         environment-file: false
-        cache-env: true
-        environment-name: TEST
```

Comment:
We could use downloads-cache instead but according to their docs it can be even slower than direct download. See https://github.com/mamba-org/provision-with-micromamba/pull/38#discussion_r808837618